ModelCatalogueCorePluginTestApp/grails-app/controllers/org/modelcatalogue/core/DataModelController.groovy

```diff
@@ -87,10 +88,6 @@ class DataModelController<T extends CatalogueElement> extends AbstractCatalogueE
         dataModelGormService.findById(id)
     }
 
-    def showAssetInAngular() {
-        redirect(url: "/#/${params.id}/asset/${params.subResourceId}")
-    }
-
```

Comment:
why has this been removed?
This is used so that people clicking from an excel link can see an item,